### ISSUE TITLE
node crash in v1.1.0

### ISSUE THREAD

**Issue by binlaniua** [oracle/node-oracledb#207]
test.js
```javascript
var oracle = require('oracledb');
oracle.createPool({
    user: 'xx',
    password: 'xx',
    connectString: 'xx'
}, function (err, pool) {
    if(pool){
        console.log(2)
        console.log(pool)
        console.log(1)
    }
    else{
        console.error(err);
    }
});
```

env:
```javascript
vmware mac 10.10.4
node 0.12.0
ide webstorm 9
```

v1.0.0 and v1.1.0
```javascript
/usr/local/bin/node test.js
2

Process finished with exit code 139
```

v.0.6.0
```javascript
/usr/local/bin/node test.js
2
{ stmtCacheSize: 30,
  connectionsInUse: 0,
  connectionsOpen: 0,
  poolTimeout: 60,
  poolIncrement: 1,
  poolMin: 0,
  poolMax: 4 }
1
```

**Comment by cjbj** [comment]
Works for me with Node 0.12.7 on OS X 10.10.5.  Do you get any Oracle traces?  Does using segfault-handler show a stack trace? Can you try a later Node.js version?

cjones@cjones-mac:~/n$ node version.js 
Driver version number is 10100
Driver version text is 1.1.0
cjones@cjones-mac:~/n$ node issue207.js 
2
{ stmtCacheSize: 30,
  connectionsInUse: 0,
  connectionsOpen: 0,
  poolTimeout: 60,
  poolIncrement: 1,
  poolMin: 0,
  poolMax: 4 }
1

**Comment by cjbj** [comment]
@binlaniua Is this still a problem?

**Comment by David-Turbert** [comment]
I think the problem is the same as #115, node V4 is not compatible

**Comment by cjbj** [comment]
@David-Turbert it is unlikely to be #115 since Node was stated to be 0.12.0.

**Comment by binlaniua** [comment]
@cjbj thanks, problem solved,  i update node to 0.12.7,  run ok
